src/app/shared/input/input.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-input',
+  templateUrl: './input.component.html',
+  styleUrls: ['./input.component.scss'],
+})
+export class InputComponent {
+  @Input() label = '';
+  name: string = '';
```

Comment:
you don't need to type variables (but you can ofc), which have simple values like boolean/string etc when you assign them value on init.

```
public name = 'Ania' // not necessary because type is obvious
public name: string // necessary ebcause value isn't known
```